### PR TITLE
cronjob. added cron to delete invalid JWT tokens

### PR DESCRIPTION
**Pull request by edospadoni**
Added cronjob to remove invalid JWT tokens.

Reference: https://trello.com/c/Z9likrdy/292-jwt-tokens-automatic-cleanup

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -107,6 +108,14 @@ func main() {
 		}))
 	})
 
+	// run expired token cleanup, on starup
```
Comment (andre8244) [inline main.go]:
```suggestion
	// run expired token cleanup, on startup
```

File: methods/auth.go [hunk 2]
```diff
@@ -390,7 +390,7 @@ func CheckTokenValidation(username string, token string) bool {
 	}
 	secrestList := string(secrestListB)
 
-	// //check whether s contains substring text
+	// check whether s contains substring text
```
Comment (andre8244) [inline methods/auth.go]:
```suggestion
	// check whether secret list contains token
```

File: methods/auth.go [hunk 3]
```diff
@@ -537,3 +539,66 @@ func UpdateRecoveryCodes(username string, codes []string) bool {
...
+
+		// check error
+		if err != nil {
+			logs.Logs.Println("[ERR][JWT] Failed to read tokens file " + configuration.Config.TokensDir + "/" + username.Name() + ". Error: " + err.Error())
+		}
+
+		// get string file
+		tokensList := string(tokenstListB)
+
+		// convert in array
```
Comment (andre8244) [inline methods/auth.go]:
```suggestion
		// convert to array
```

**Review by andre8244 — APPROVED** [review]
Just a couple of typos

**Comment by edospadoni** [comment]
> Just a couple of typos

thanks @andre8244 :pray:

**Pull request merged**